### ISSUE TITLE
Document ES7 configuration with babel for ava

### ISSUE THREAD

**Issue by talexand** [sindresorhus/ava#513]
The documentation includes information on how to get ES6 support for ava, which is great. Would it be possible to include docs on how to get ES7 in also?

Here's a realistic use case:

A React component with the `@connect()` decorator from Redux:

```js
import React from 'react';
import { connect } from 'react-redux';

@connect()
class Foo extends React.Component {

}
```

A test file for unit testing it:

```js
import test from 'ava';
import { isElementOfType } from 'react-addons-test-utils';
import 'babel-core/register'; // as per docs

var Foo = require('../src/foo.jsx');

test('foo ', t => {
  const simpleFoo = React.createElement(Foo);
  t.ok(isElementOfType(Foo, simpleFoo));
});
```

I tried using [a shim that apparently worked for people back when iojs was separate](http://www.jbernier.com/how-to-get-es7-async-functions-with-babel), but the namespaces don't match anymore:

```js
require("babel/register")({
  optional: ['es7.asyncFunctions']
});
```

On the other hand, [this guy](https://glebbahmutov.com/blog/picking-javascript-testing-framework/) claims ava supports es7 without any extra effort.

Either way, it would be nice to get some documentation on this--I'm interested in using ava going forward.

**Comment by talexand** [comment]
This didn't work either:

```
require("babel-register")({
  optional: ["es7.decorators"]
});
```

**Comment by Whoaa512** [comment]
AVA used to implicitly read in your local .babelrc file before 0.10 (See #398) which would allow you to use es7 or any arbitrary babel plugins. 

See #448 to contribute to the discussion on how customizable babel options should be handled.

**Comment by sindresorhus** [comment]
Yes, we need to resolve this soon. We need more feedback on #448 ;)

**Comment by novemberborn** [comment]
Closing this in favor of #448, under the assumption that the implementation would be obvious once that lands.
